src/main/resources/com/puppycrawl/tools/checkstyle/grammars/javadoc/JavadocLexer.g4

```diff
@@ -120,7 +120,8 @@ Newline5: NEWLINE
 Leading_asterisk3: LEADING_ASTERISK -> type(LEADING_ASTERISK);
 XmlTagOpen1: '<' -> type(OPEN), pushMode(xmlTagDefinition);
 STRING: '"' .*? '"' {referenceCatched = false;} -> mode(DEFAULT_MODE);
-PACKAGE: [a-z_$] ([a-z0-9_$] | '.')+ [a-z0-9_$] {referenceCatched = true;};
+PACKAGE: [a-z_$] ([a-z0-9_$] | '.')+ [a-z0-9_$] {_input.LA(1) == '.' || _input.LA(1) == '}'
+                     || Character.isWhitespace(_input.LA(1))}? {referenceCatched = true;};
```

Comment:
@ps-sp , `@see` tag has exact forms by specification - http://docs.oracle.com/javase/8/docs/technotes/tools/windows/javadoc.html#CHDDIEDI : 
`@see string`
`@see <a href="URL#value">label</a>`
`@see package.class#member label`

your example should be parsed as `@see string`
grammar is enough to make that parsing.

@Vladlis , please review this one more time. I do not understand why this change is required.